src/hapi/pipelines/utilities/orgtype.py

```diff
@@ -0,0 +1,39 @@
+import logging
```

Comment:
I would call this org_type.py just to be consistent with the class and also DB table name